Apps/AdminWebClient/src/ClientApp/src/views/Dashboard.vue

```diff
@@ -653,4 +631,16 @@ export default class Dashboard extends Vue {
         padding: 0px;
     }
 }
+
+.refresh-button {
+    position: fixed;
+    right: 16px;
+    top: 80px;
+    z-index: 1000;
```

Comment:
Instead of doing custom positioning it could use the vuetify grid and/or the spacing classes

Reply:
I made it a floating action button with a fixed position so that it can be seen even when scrolling down the page. The custom CSS seemed to be the easiest way to do this with Vuetify.

Reply:
fair enough, I thought vuetify had similar features to bootstrap for positioning